### PR TITLE
[Homework #3] - HTML by Saidahmad Kholmuhammadkhonov

### PR DESCRIPTION
**Pull request by Saidahmadkhon001**

### CODE REVIEW

File: homeworks/saidahmad-kholmuhammadkhonov/03-html/contact.html [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+        <a href="home.html">Home</a>
+        <a href="contact.html">Contact Me</a>
+        <hr>
+    </nav>
+    <header>
+            <h1>Contact Me</h1>     
+    </header>
+    <main>
+        <form action="" method="post">
+            <label for="">
```
Comment (VitalikMelnyk) [inline homeworks/saidahmad-kholmuhammadkhonov/03-html/contact.html:20]: Connect label with input.

File: homeworks/saidahmad-kholmuhammadkhonov/03-html/contact.html [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+                Full name*
+                <input type="text" name="firstname" placeholder="Your name" required/>
+                <input type="text" name="lastname" placeholder="Your last name" required/>
+            </label>
+            <br>
+            <label for="">
```
Comment (VitalikMelnyk) [inline homeworks/saidahmad-kholmuhammadkhonov/03-html/contact.html:26]: Connect label with input.

File: homeworks/saidahmad-kholmuhammadkhonov/03-html/home.html [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+        <a href="contact.html">Contact Me</a>
+        <hr>
+    </nav>
+    <header>
+            <h1>Hi, I am Saidahmad, Front-End Developer</h1>     
+    </header>
+    <main>
+        <p>I am eager to learn and grow as a developer, bringing a creative and 
+            solution-focused approach to every project.</p>
+        <img src="/images/profile_photo.jpg" alt="Profile photo" width="250px">
```
Comment (VitalikMelnyk) [inline homeworks/saidahmad-kholmuhammadkhonov/03-html/home.html:21]: The path to the image not working.

File: homeworks/saidahmad-kholmuhammadkhonov/03-html/home.html [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+    </header>
+    <main>
+        <p>I am eager to learn and grow as a developer, bringing a creative and 
+            solution-focused approach to every project.</p>
+        <img src="/images/profile_photo.jpg" alt="Profile photo" width="250px"> 
+        <br>
+        <a href="/files/SaidahmadFrontEndDev.pdf" download="Resume">Download Resume</a>  
+        <hr> 
+    </main>
+    <footer>
```
Comment (VitalikMelnyk) [inline homeworks/saidahmad-kholmuhammadkhonov/03-html/home.html:26]: Add alt text to each icon.

File: homeworks/saidahmad-kholmuhammadkhonov/03-html/home.html [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+    </nav>
+    <header>
+            <h1>Hi, I am Saidahmad, Front-End Developer</h1>     
+    </header>
+    <main>
+        <p>I am eager to learn and grow as a developer, bringing a creative and 
+            solution-focused approach to every project.</p>
+        <img src="/images/profile_photo.jpg" alt="Profile photo" width="250px"> 
+        <br>
+        <a href="/files/SaidahmadFrontEndDev.pdf" download="Resume">Download Resume</a>
```
Comment (VitalikMelnyk) [inline homeworks/saidahmad-kholmuhammadkhonov/03-html/home.html:23]: The same with file. Not found.

File: homeworks/saidahmad-kholmuhammadkhonov/03-html/contact.html [hunk 6]
```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
```
Comment (VitalikMelnyk) [inline homeworks/saidahmad-kholmuhammadkhonov/03-html/contact.html:1]: Check all paths to images and icons.

**Review by VitalikMelnyk — COMMENTED** [review]
Inside assets make some changes:
1. Divide into icons and images
2. Move items to the appropriate folder.
3. Fix the path to each item in the HTML file.

**Review by solvd-mmelnyk — APPROVED** [review]
Looks good!

**Pull request merged**